packages/cli/templates/webcomponents/igc-ts/projects/empty/files/package.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Format each one of the files in the webcomponents folder (Shift + Alt + F), so the .editorconfig rules are applied.